src/tools/annotation/bidirectionalTool/moveHandle/moveHandle.js

```diff
@@ -50,6 +51,33 @@ export default function(
       EVENTS.MEASUREMENT_MODIFIED,
       modifiedEventData
     );
+
+    const measurementModifiedHandler = () => {
```

Comment:
I think the current `moveHandle` (generic) accomplishes this by setting the tool's annotation data to `invalidated`. Then, in the render loop, if we see `invalidated`, we can run the calculate method to grab updated data and inform measurement modified subscribers.

This has the advantage of only needing to call the  method in one place.

Reply:
I have done a small refactor, I am now just calculating and triggering the measurement modified. 
The measurement table uses this measurement modified to update the table. I will think later on something cleaner for other annotationTools.